app/models/concerns/participant_record_event_recorder.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Currently a placeholder to ensure that transitioning to this doesn't break since the scopes will be different
+# 1. Duplicate ect_profiles can be recorded, so we need to filter for uniqueness as a count
+# 2. Order is reversed, to pick up the latest change to a user
+# 3. Holding "retention_1" rather than "Retention 1" in the db, for example.
+
+module Concerns
+  module ParticipantRecordEventRecorder
```

Comment:
This is something that I was doing with feature flags and making sure that it could work with the changes that are proposed to how we are going to record events rather than state transitions. Probably should go into a separate commit on a different ticket.